.github/ISSUE_TEMPLATE/cut_release.md

```diff
@@ -34,15 +37,22 @@ To find more information on specific steps, please see documentation [here](http
 - [ ] Release Captain: Update RKE2
 - [ ] Release Captain: Tag new RKE2 RC
 - [ ] Release Captain: Tag new RKE2 packaging RC "testing"
+- [ ] Release Captain: Prepare PRs as needed to update [KDM](https://github.com/rancher/kontainer-driver-metadata/) in the appropriate dev branches using an RC.  For more information on the structure of the PR, see the [docs](https://github.com/rancher/rke2/blob/master/developer-docs/upgrading_kubernetes.md#update-rancher-kdm)
+  - [ ] If server args, agent args, or charts are changed, link relevant rancher/rancher issue or create new rancher/rancher issue
```

Comment:
Is this saying that we need to create rancher/rancher issue for each change to K3s or RKE2 args/charts?

Reply:
If we are exposing those changes in Rancher through KDM, yes.

Reply:
*who* does it can be debated, but its ultimately on the rke2 team to do so

Reply:
Does this need to be distinct from the version bump issue, or can we use the same for both?